nixpkgs_review/utils.py

```diff
@@ -26,7 +26,10 @@ def wrapper(text: str) -> None:
 warn = color_text(31, file=sys.stderr)
 info = color_text(32)
 skipped = color_text(33)
-link = color_text(34)
+
+
+def link(uri: str) -> None:
+    color_text(34)(f"\u001b]8;;{uri}\u001b\\{uri}\u001b]8;;\u001b\\\n")
```

Comment:
The other PR only makes it a link if it is a TTY. This one should do the same.

Reply:
Sent this separately to fix the existing link: https://github.com/Mic92/nixpkgs-review/pull/489